roles/grafana/tasks/configure-slack.yml

```
@@ -32,5 +32,41 @@
   with_items:
     "{{ kafka3_variables }}"
   when:
-    ("slack_notification" not in alert_list)
+    ("slack_infrastructure_notification" not in alert_list)
```

The parentheses here aren't necessary as we're not grouping any conditions. Also this should be written as a list item:

```suggestion
    - "slack_infrastructure_notification" not in alert_list
```

Or alternatively collapse the `when` statement and condition into a single line:

```
when: "slack_infrastructure_notification" not in alert_list
```